includes/plugins/class-nextgen.php

```diff
@@ -102,8 +102,10 @@ private function find_images( $gallery_id, $gallery_path ) {
 
                 $data = array(
                     'source_url' => $source_url,
-                    'caption' => $nextgen_image->description,
-                    'alt' => $nextgen_image->alttext,
+                    'slug' => $nextgen_image->filename,
+                    'caption' => $nextgen_image->alttext,
```

Comment:
Just checking this logic - we want to map the nextgen alt text to the attachment caption?